internal/controllers/objecttemplate/template_reconciler.go

```diff
@@ -106,10 +112,26 @@ func (r *templateReconciler) Reconcile(
 	obj.SetAnnotations(labels.Merge(existingObj.GetAnnotations(), obj.GetAnnotations()))
 
 	obj.SetResourceVersion(existingObj.GetResourceVersion())
-	if err := r.client.Update(ctx, obj); err != nil {
-		return res, fmt.Errorf("updating templated object: %w", err)
+	if err := r.fixFieldManagers(ctx, existingObj); err != nil {
```

Comment:
On second thought this might not be required since PKO creates/updates these objects using not create and update calls. No CSA apply fields would be present. wdyt @erdii